src/pages/SubmissionConfig.js

```diff
@@ -245,115 +245,146 @@ export const SubmissionConfig = ({ methodId }) => {
+            h(TextInput, {
+              disabled: isSubmitting,
+              'aria-label': 'Submission name',
+              value: runSetName,
+              onChange: setRunSetName,
+              placeholder: 'Enter submission name'
+            })
+          ]
+          ),
+          div({ style: { lineHeight: 2.0, marginTop: '1.5rem' } }, [
+            span({ style: { fontSize: 16, fontWeight: 'bold' } }, ['Comment ']), '(optional)',
+            h(TextArea, {
+              style: { height: 200, borderTopLeftRadius: 0, borderTopRightRadius: 0 },
+              'aria-label': 'Enter a comment',
+              disabled: isSubmitting,
+              value: runSetDescription,
+              onChange: setRunSetDescription,
+              placeholder: 'Enter comments'
+            })
+          ]),
+          div({ style: { lineHeight: 2.0, marginTop: '1.5rem' } }, [
+            div([h(TextCell, ['This will launch ', span({ style: { fontWeight: 'bold' } }, [_.keys(selectedRecords).length]), ' workflow(s).'])]),
+            h(TextCell, { style: { marginTop: '1rem' } }, ['Running workflows will generate cloud compute charges.'])
+          ])
+        ]),
+        viewWorkflowScriptModal && h(ViewWorkflowScriptModal, { workflowScript, onDismiss: () => setViewWorkflowScriptModal(false) })
       ]),
-      viewWorkflowScriptModal && h(ViewWorkflowScriptModal, { workflowScript, onDismiss: () => setViewWorkflowScriptModal(false) })
+      div({ style: { marginRight: '1em' } }, [
+        div({ style: { backgroundColor: colors.accent(0.2), paddingTop: '0.25em', paddingBottom: '0.25em', paddingLeft: '1em', paddingRight: '1em' } }, [
+          h4('Have questions?'),
+          method && isCovid19Method(method.name) && p([
+            h(Link, { href: 'https://support.terra.bio/hc/en-us/articles/12028928980123-Covid-19-Surveillance-tutorial-guide', ...Utils.newTabLinkProps },
+              [
+                'Covid-19 Surveillance tutorial guide',
+                icon('pop-out', { size: 12, style: { marginLeft: '0.25rem' } })
+              ]
+            )
+          ]),
+          method && isCovid19Method(method.name) && p([
+            h(Link, { href: 'https://app.terra.bio/#workspaces/azure-featured-workspaces/COVID-19-Surveillance', ...Utils.newTabLinkProps },
+              [
+                'Covid-19 Featured Workspace',
+                icon('pop-out', { size: 12, style: { marginLeft: '0.25rem' } })
+              ]
+            )
+          ]),
+          p([
+            h(Link, { href: 'https://support.terra.bio/hc/en-us/articles/12029178977307-How-to-set-up-and-run-a-workflow', ...Utils.newTabLinkProps },
+              [
+                'How to set up and run a workflow',
+                icon('pop-out', { size: 12, style: { marginLeft: '0.25rem' } })
+              ]
+            )
+          ])
+        ])
+      ])
```

Comment:
L359-L387 is the main change in this file. The changes above these lines are because I needed to wrap the workflow details in a separate div to create flex columns.

Reply:
thanks for the callout!

Reply:
this actually feels like it could be its own component... even if it's a small one. What do you think?

Reply:
The links box has been refactored into a component.